src/Illuminate/Database/Schema/Builder.php

```diff
@@ -325,6 +325,27 @@ public function dropColumns($table, $columns)
         });
     }
 
+    /**
+     * Drop index from a table schema only if exists.
+     *
+     * @param  string  $table
+     * @param  string  $index
```

Comment:
Why not introduce the capability to drop more than one index at the same time? By considering all Laravel ecosystem maybe it can be a good addition here.

Reply:
@devajmeireles good idea. i modified my code to also accept array of indexes as well.